Packs/Palo_Alto_Networks_WildFire/Integrations/Palo_Alto_Networks_WildFire_v2/Palo_Alto_Networks_WildFire_v2.yml

```diff
@@ -60,7 +71,7 @@ script:
       description: File hash to check.
       isArray: true
       name: file
-      required: false
+      required: true
```

Comment:
this is the reason for the BC break, changing this is a must? or we can leave it false?